source/cloud-security/azure/activity-services/prerequisites/dependencies.rst

```diff
@@ -6,73 +6,22 @@
 Installing dependencies
 =======================
 
-.. note::
-
-  The Azure monitoring module can be configured in the Wazuh manager (which also behaves as an agent) or directly in a Wazuh agent.
-
-.. warning::
-  The Wazuh manager includes all dependencies installed, these steps are only necessary when configuring the integration in a Wazuh agent.
+.. |service| replace:: Azure
 
+.. include:: /_templates/cloud/notes.rst
 
 Python
 ------
 
-The Azure module requires `Python 3 <https://www.python.org/>`__. Specifically, it's compatible with
-`Python |PYTHON_CLOUD_CONTAINERS_MIN|–|PYTHON_CLOUD_CONTAINERS_MAX| <https://www.python.org/downloads/>`_. While later Python versions should work as well, we can't assure they are compatible.
-
-.. tabs::
-
-   .. group-tab:: Yum
-
-      .. code-block:: console
-
-         # yum update && yum install python3
-
-   .. group-tab:: APT
-
-      .. code-block:: console
-
-         # apt-get update && apt-get install python3
-
-
-The required modules can be installed with Pip, the Python package manager. Most of UNIX distributions have this tool available in their software repositories:
-
-.. tabs::
-
-   .. group-tab:: Yum
-
-      .. code-block:: console
-
-         # yum update && yum install python3-pip
-
-   .. group-tab:: APT
-
-      .. code-block:: console
-
-         # apt-get update && apt-get install python3-pip
+.. |service_module| replace:: ``Azure``
```

Comment:
We could remove this replacement and simply use `|service|` if they get unified.